tox.ini

```diff
@@ -33,3 +33,8 @@ ignore = E402
 [pytest]
 norecursedirs = .tox .git .hg sandbox build
 python_files = test_*.py
+filterwarnings =
+    default::DeprecationWarning
+    default::PendingDeprecationWarning
+    ignore:.*inspect.getargspec.* is deprecated.*
+    ignore:invalid escape sequence.*
```

Comment:
@allan-silva sorry for the delay in getting back to this. We've taken the pytest 3.9 upgrade via pyup so it would be good to rerun the tests and make sure the filters are still set correctly. 

I also wondered if this approach might be helpful in ./tox.ini:
```
filterwarnings =
    ignore
    error:::auslib
    default::DeprecationWarning:auslib
    default::PendingDeprecationWarning:auslib
```
The idea is to ignore all warnings in deps, while in auslib warn about deprecations, and upgrade to error anything else. It would avoid suppressing the named errors in Balrog code, I think.

The other tox.ini would need equivalent changes too ?